### ISSUE TITLE
Use larger-icons on some pokemons

### ISSUE THREAD

**Issue by DavidSelem** [modrzew/pokeminer#138]
Used to just copy and rewrite the images from icons folder using the icons from larger-icons, but now the icon size is fixed at newmap.html ( iconSize: [30, 30],)

I commented that line but the icons are so inaccurate, specially the larger icons

Could there be a new config line (like trash_id) to specify which pokemons are "important" so they are bigger than usual?

**Comment by modrzew** [comment]
There's no such config yet. You'd need to create a "class" similar to PokemonIcon with different properties (like size), and then place similar if in `PokemonMarker`.

**Comment by DavidSelem** [comment]
I wrote this:         
var BigPokemonIcon = L.Icon.extend({
            options: {
               iconSize: [80, 60],
                popupAnchor: [0, -16]
            }
        });

if (raw.pokemon_id == 10){
            var bigicon = new BigPokemonIcon({iconUrl: '/static/larger-icons/' + raw.pokemon_id + '.png'});
		}

Is raw.pokemon_id the ideal way to get the ID ?
I know I need to compare IDs to get working this "feature"

**Comment by modrzew** [comment]
Yeah, raw.pokemon_id contains Pokemon number. Then just use it in a marker:

```js
var marker = L.marker([raw.lat, raw.lon], {icon: bigicon, opacity: 1});
```

**Comment by DavidSelem** [comment]
Done, thank you! http://i.imgur.com/pydsHmR.png
